### PR TITLE
feat: add get delete owners

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
get list of owners for a bucket
add owner to a bucket
delete owner from a bucket

### CODE REVIEW

File: src/common/paths/buckets_bucketID_owners.yml [hunk 1]
```diff
@@ -29,31 +55,77 @@ post:
...
         schema:
           $ref: "../schemas/AddResourceMemberRequestBody.yml"
+        examples:
+          $ref: "../responses/example-responses/PostBucketsIDOwnersSuccessResponse.yml"
   responses:
     "201":
-      description: Success. The user is an owner of the bucket
+      description: |
+        Success.
+        The response body contains the user information.
```
Comment (jstirnaman) [inline src/common/paths/buckets_bucketID_owners.yml:95]:
```suggestion
        Created. The bucket `owner` role is assigned to the user.
        The response body contains the resource owner with
        role and user detail.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Small fixes and suggestions. Trying to make it consistent with task owners.

File: src/common/paths/buckets_bucketID_owners.yml [hunk 2]
```diff
@@ -29,31 +55,77 @@ post:
   tags:
     - Buckets
   summary: Add an owner to a bucket
+  description: |
+    Add a owner to a bucket and return the new owner information.
```
[the remark this answers is not in the document]
Comment (jstirnaman) [reply src/common/paths/buckets_bucketID_owners.yml:59]: I suggest getting kelsiev's input here.

File: src/common/paths/buckets_bucketID_owners.yml [hunk 3]
```diff
@@ -29,31 +55,77 @@ post:
...
+
+    InfluxDB owners have permission to delete resources and other users from
+    the resource.
+
+    Use this endpoint to give `owner` privileges to a bucket.
+
+    #### InfluxDB Cloud
+
+    - InfluxDB Cloud has only one permission level for users: `owner`.
+
```
Comment (jstirnaman) [inline src/common/paths/buckets_bucketID_owners.yml:69]:
```suggestion
```
Now that I know owners are only relevant for orgs and buckets, I think we should describe them specifically for buckets here.

File: src/common/paths/buckets_bucketID_owners_userID.yml [hunk 4]
```diff
@@ -3,23 +3,49 @@ delete:
...
+
+    Use this endpoint to remove an owner from a bucket. This
+    removes the user's `owner` permissions for the bucket. With `owner`
+    permissions, a user can delete resources and other users from your
+    organization.
+
+    #### InfluxDB Cloud
+
+    - InfluxDB Cloud has only one permission level for users: `owner`.
+
```
Comment (jstirnaman) [inline src/common/paths/buckets_bucketID_owners_userID.yml:17]:
```suggestion
```

Comment (jstirnaman) [reply src/common/paths/buckets_bucketID_owners_userID.yml:17]: Maybe describe how `role: owner` differs from auths?

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Sorry my feedback here is a hot mess. Basically, I'm retracting my previous comments since learning more about owners (and members). I think we should be specific about what they can do with buckets and buckets users (and members), and similarly with orgs.

File: src/common/paths/buckets_bucketID_members.yml [hunk 5]
```diff
@@ -103,7 +103,7 @@ post:
       $ref: "../responses/BadRequestError.yml"
       examples:
         invalidRequest:
-          summary: The `userID` is missing from the request body.
+          summary: The user `id` is missing from the request body.
```
Comment (jstirnaman) [inline src/common/paths/buckets_bucketID_members.yml:106]: 👍

File: src/common/paths/buckets_bucketID_owners.yml [hunk 6]
```diff
@@ -3,21 +3,64 @@ get:
...
+    #### InfluxDB Cloud
+
+    - Doesn't use `owner` and `member` roles.
+      Use [`/api/v2/authorizations`](#tag/Authorizations) to assign user permissions.
+
+    #### Limitations
+
+    - Owner permissions are separate from API token permissions.
+    - Owner permissions are used in the context of the InfluxDB UI.
```
Comment (jstirnaman) [inline src/common/paths/buckets_bucketID_owners.yml:21]: 👍

File: src/common/paths/buckets_bucketID_owners.yml [hunk 7]
```diff
@@ -3,21 +3,64 @@ get:
...
+
+    #### Required permissions
+
+    - `read-orgs INFLUX_ORG_ID`
+    `INFLUX_ORG_ID` is the ID of the organization that you want to retrieve a
```
Comment (jstirnaman) [inline src/common/paths/buckets_bucketID_owners.yml:26]:
```suggestion

    `INFLUX_ORG_ID` is the ID of the organization that you want to retrieve a
```

File: src/common/paths/buckets_bucketID_owners.yml [hunk 8]
```diff
@@ -29,31 +72,96 @@ post:
   tags:
     - Buckets
   summary: Add an owner to a bucket
+  description: |
+    Adds an owner to a bucket and returns the [owners]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#owner)
+    with role and user detail.
+
+    Assigns a bucket `owner` role to a user and returns the new owner detail.
```
Comment (jstirnaman) [inline src/common/paths/buckets_bucketID_owners.yml:79]: Can we combine these sentences?

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: src/common/paths/buckets_bucketID_owners.yml (hunk 8 above)
Comment (sunbryely-influxdata) [reply src/common/paths/buckets_bucketID_owners.yml:79]: I missed this and actually meant to remove the second sentence as it has a duplicate meaning. Thanks for catching this

**Review by jstirnaman — APPROVED** [review]

**Pull request merged**